plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/DefaultedPipelineOptionsComponentTest.java

```diff
@@ -158,7 +157,7 @@ public void testCustomValues_nulls() {
 
     // the empty/null customValues should override the preferences
```

Comment:
Comment doesn't fit with the line below it anymore. Should `testCustomValues_nulls` be providing an email?